NoClassDefFoundError (AnvilGUI$Builder)

Hello i have Caused by: java.lang.NoClassDefFoundError: net/wesjd/anvilgui/AnvilGUI$Builder, i have try "use mvn clean package command to build the uber jar." but it doesnt work.

I have the same issue. Tested whithe mvn clean install (and clean package).

on **net.minecraft.server.v1_15_R1**

To complete the message above : 

```
Caused by: java.lang.ClassNotFoundException: net.wesjd.anvilgui.AnvilGUI$Builder
at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_242]
at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:171) ~[patched_1.15.2.jar:git-Paper-201]
at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100) ~[patched_1.15.2.jar:git-Paper-201]
at java.lang.ClassLoader.loadClass(ClassLoader.java:419) ~[?:1.8.0_242]
at java.lang.ClassLoader.loadClass(ClassLoader.java:352) ~[?:1.8.0_242]
... 23 more
```

Thanks for that works, and thanks for your futures answers

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ on the issues tab.
 ## How to use
 
 AnvilGUI requires the usage of Maven or a Maven compatible build system. 
-You can find the repository and dependency information [here](https://jitpack.io/#WesJD/AnvilGUI).
+You can find the repository and dependency information [here](https://jitpack.io/#WesJD/AnvilGUI/master-SNAPSHOT).
 
 ### In your plugin
 

